### ISSUE TITLE
Github search is broken

### ISSUE THREAD

**Issue by soehlert** [sherlock-project/sherlock#175]
I have two different github accounts and it found neither of them. Both have plenty of public repos too.

**Comment by hoadlck** [comment]
There have been problems with GitHub in the past, but the problems should be fixed.  I am seeing your name detected properly on my PC.
```
(511)% python -u sherlock.py --site github soehlert
                                              ."""-.
                                             /      \
 ____  _               _            _        |  _..--'-.
/ ___|| |__   ___ _ __| | ___   ___| |__    >.`__.-""\;"`
\___ \| '_ \ / _ \ '__| |/ _ \ / __| |/ /   / /(     ^\
 ___) | | | |  __/ |  | | (_) | (__|   <    '-`)     =|-.
|____/|_| |_|\___|_|  |_|\___/ \___|_|\_\    /`--.'--'   \ .-.
                                           .'`-._ `.\    | J /
                                          /      `--.|   \__/

[*] Checking username soehlert on:
[+] GitHub: https://www.github.com/soehlert                                                                                                                                                                                                   
```
What version of Sherlock are you using?  I am using 0.5.3.

Is there anything odd about your connection? Do you use a proxy?

**Comment by soehlert** [comment]
Interesting. Running it from docker here is what I see.

```
$ docker run theyahya/sherlock soehlert
                                               ."""-.
                                              /      \
 ____  _               _            _        |  _..--'-.
/ ___|| |__   ___ _ __| | ___   ___| |__    >.`__.-""\;"`
\___ \| '_ \ / _ \ '__| |/ _ \ / __| |/ /   / /(     ^\
 ___) | | | |  __/ |  | | (_) | (__|   <    '-`)     =|-.
|____/|_| |_|\___|_|  |_|\___/ \___|_|\_\    /`--.'--'   \ .-.
                                           .'`-._ `.\    | J /
                                          /      `--.|   \__/

[*] Checking username soehlert on:
[+] Instagram: https://www.instagram.com/soehlert
[+] Twitter: https://www.twitter.com/soehlert
[-] Facebook: Not Found!
[+] YouTube: https://www.youtube.com/soehlert
[-] Blogger: Not Found!
[-] Google Plus: Not Found!
[+] Reddit: https://www.reddit.com/user/soehlert
[+] Pinterest: https://www.pinterest.com/soehlert
[-] GitHub: Not Found!
...

**Comment by nstapelbroek** [comment]
I'm able to reproduce this issue. On a local clone the Github search works and searching with the docker image yields no results. It seems that the image itself has not been updated for a while. Maybe we can make it auto-build on every merge to the `master` branch?

**Comment by sdushantha** [comment]
Since this got fixed in #201, I am closing this :)
